src/contracts/proxies/actions/BasicActions.sol

```diff
@@ -299,14 +299,7 @@ contract BasicActions is CommonActions, IBasicActions {
     );
 
     // Paybacks debt to the SAFE (allowed because reducing debt of the SAFE)
-    ISAFEEngine(_safeEngine).modifySAFECollateralization({
-      _cType: _safeInfo.collateralType,
-      _safe: _safeInfo.safeHandler,
-      _collateralSource: address(this),
-      _debtDestination: address(this),
-      _deltaCollateral: 0,
-      _deltaDebt: -int256(_safeData.generatedDebt)
-    });
+    _modifySAFECollateralization(_manager, _taxCollector, _safeId, 0, -_safeData.generatedDebt.toInt());
```

Comment:
Is `_sData.safeHandler` (what it is now) in place of `address(this)` (what it was previously) for `_collateralSource` and `_debtDestination` correct here?